test/unit/lib/plugins/aws/package/lib/mergeIamTemplates.test.js

```diff
@@ -880,6 +259,13 @@ describe('lib/plugins/aws/package/lib/mergeIamTemplates.test.js', () => {
                 handler: 'index.handler',
                 disableLogs: true,
               },
+              fnWithVpc: {
+                handler: 'func.function.handler',
```

Comment:
Doesn't that break? I think this handler is pointing not existing file. Why it's not just `index.handler` (in context of this test fact that all lambdas point same files is not that relevant)